rule/fetcher_default.go

```diff
@@ -190,31 +187,26 @@
 }
 
 func (f *FetcherDefault) watch(ctx context.Context, watcher *fsnotify.Watcher, events chan event) error {
-	var pc map[string]interface{}
-
-	viperx.AddWatcher(func(e fsnotify.Event) error {
-		if reflect.DeepEqual(pc, viper.Get(configuration.ViperKeyAccessRuleRepositories)) {
-			f.r.Logger().
-				Debug("Not reloading access rule repositories because configuration value has not changed.")
-			return nil
+	f.config.AddWatcher(func(e watcherx.Event, err error) {
+		if err != nil {
```

Comment:
You removed the check that detects whether the value has actually changed. Is that intentional?

Reply:
Yea that code path never triggered because the cached value was never set.